marshal/consumer.go

```diff
@@ -458,76 +485,59 @@ func (c *Consumer) claimTopics() {
 	}
 }
 
-// updatesTopicClaims if changed. It should be noted that it expects c.lock.RLock
-// to be already acquired
-func (c *Consumer) safeUpdateTopicClaims(latestClaims map[string]bool, force bool) {
```

Comment:
I think it depends on what you're trying to communicate. The current definition, it means, topic-claims have been updated. If we're just sending periodic notification, that's no longer true. We need to double check code that is using the channel to indicate changes as opposed to periodic update and make sure they are doing the diff themselves (if they care). I will have to double check Cape's code at least.